pkgs/development/python-modules/pyprof2calltree/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, buildPythonApplication, fetchPypi }:
+
+buildPythonApplication rec {
+  name = "${pname}-${version}";
```

Comment:
no `name`

Reply:
Should this just be left out entirely? Is it automatically generated?
Apologies for any stupid questions - this is my first contribution.

Reply:
Yes. Yes it is.